static/styles/_quitGame.scss

```diff
@@ -0,0 +1,26 @@
+.quit-game {
+  //   display: inline;
```

Comment:
Clean the unused code, organize properties and add pointer to the cursor please.